File: 1490-generate-a-string-with-characters-that-have-odd-counts/generate-a-string-with-characters-that-have-odd-counts.py
class Solution:
    def generateTheString(self, n: int) -> str:
        s=""
        f=1
        if n%2==0:
            c=n-1
            s+="a"*c
            f=0
        else:
            s+="a"*n
        if f==0:
            s+="b"
        return s
        

File: 1490-generate-a-string-with-characters-that-have-odd-counts/generate-a-string-with-characters-that-have-odd-counts.java
class Solution {
    public String generateTheString(int n) {
        String ans="a";
        if(n%2!=0)
        {
            ans= ans.repeat(n);
            return ans;
        }
        else
        {
            ans= ans.repeat(n-1);
            return ans+"b";

        }
        
    }
}